rocky/assets/css/components/table.scss

```diff
@@ -81,3 +82,11 @@ table tr.expando-row td {
     padding: var(--table-cell-padding);
   }
 }
+
+table tbody td {
```

Comment:
When introducing this, we should probably also add valign:top to the other cells to provide a more consistent expderience for the remainder of the table-row if multi-line wrapping occurs .